src/main/java/RacingCarGameResultView.java

```diff
@@ -1,31 +1,46 @@
 import java.util.List;
+import java.util.Map;
 
 public class RacingCarGameResultView {
 
   public static final String PROGRESS_SYMBOL = "-";
+  public static final String DELIMITER = Constants.STRING_DELIMITER + " ";
 
-  private RacingCarGame racingCarGame;
+  private RacingCarGameResults racingCarGameResults;
 
-  public RacingCarGameResultView(RacingCarGame racingCarGame) {
-    this.racingCarGame = racingCarGame;
+  public RacingCarGameResultView(RacingCarGameResults racingCarGameResults) {
+    this.racingCarGameResults = racingCarGameResults;
   }
 
   public void showResult() {
-    List<List<Integer>> resultList = racingCarGame.getResults().getResultList();
-    resultList.forEach(result -> {
+    Map<Integer, List<RacingCarResult>> results = racingCarGameResults.getResults();
+    results.forEach((round, result) -> {
       showProgress(result);
       System.out.println();
     });
+    showWinnerNames(racingCarGameResults);
   }
 
-  private void showProgress(List<Integer> result) {
-    result.forEach(progress -> {
-      String progressExpression = "";
-      for (int i = 0; i < progress; i++) {
-        progressExpression += PROGRESS_SYMBOL;
-      }
+  private void showWinnerNames(RacingCarGameResults racingCarGameResults) {
+    List<String> winnerNames = racingCarGameResults.getWinnerNames();
+    String joinedWinnerNames = Strings.join(winnerNames, DELIMITER);
+    System.out.printf("%s가 최종 우승했습니다.%n", joinedWinnerNames);
+  }
+
+  private void showProgress(List<RacingCarResult> roundResults) {
+    roundResults.forEach(result -> {
+      System.out.print(String.format("%s : ", result.getName()));
```

Comment:
다음과 같이 해볼 수도 있을 것 같아요! 😄 
```suggestion
      System.out.printf("%s : ", result.getName());
```